### PR TITLE
#665: Add default values for strategy type & recommended label

### PR DESCRIPTION
**Pull request by oznasi1**

### CODE REVIEW

File: src/components/strategies/create/CreateStrategyTypeMenu.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const BlockIconTextDesc = ({
       </div>
       <div className={'flex-shrink space-y-6'}>
         <div className={'text-14 font-weight-500'}>{title}</div>
-        <div className={'h-32 text-12 text-white/60'}>{description}</div>
```
Comment (pingustar) [inline src/components/strategies/create/CreateStrategyTypeMenu.tsx:33]: why remove the hight class? Now the UI is broken - the icon and buttons jump around when switching between "Recurring" and "Disposable"

File: src/components/strategies/create/CreateStrategyTypeMenu.tsx [hunk 2]
```diff
@@ -49,6 +49,16 @@ export const CreateStrategyTypeMenu: FC<UseStrategyCreateReturn> = ({
     selectedTabItems,
   } = useCreateStrategyTypeMenu(base?.address!, quote?.address!, strategyType);
 
+  useEffect(() => {
```
Comment (pingustar) [inline src/components/strategies/create/CreateStrategyTypeMenu.tsx:52]: not sure we need an effect for this. Why cant you set this as default value in the state?

**Review by pingustar — CHANGES_REQUESTED** [review]

File: src/components/strategies/create/CreateStrategyTypeMenu.tsx (hunk 2 above)
Comment (oznasi1) [reply src/components/strategies/create/CreateStrategyTypeMenu.tsx:52]: We have some case that we need the useEffect for:
1. user open the page, recurring as default and  first item selected -> click again on Recurring -> the first item won't be selected.
2. user open the page, recurring as default and  first item selected -> click on 2 Ranges -> click on Recurring again the first item won't be selected.

Update me if its not clear so we can have a short discord :)

**Review by ashachaf — APPROVED** [review]

**Review by pingustar — APPROVED** [review]
LGTM

**Pull request merged**